src/models/position/batch.js

```diff
@@ -104,6 +104,37 @@ const updateParticipantPosition = async (trx, participantPositionId, participant
     })
 }
 
+const getTransferInfosToChangePosition = async (trx, transferIds, transferParticipantRoleTypeId, ledgerEntryTypeId) => {
+  try {
+    const knex = await Db.getKnex()
+    const transferInfos = await knex('transferParticipant')
+      .transacting(trx)
+      .innerJoin('transferStateChange AS tsc', 'tsc.transferId', 'transferParticipant.transferId')
```

Comment:
I think the main reason for the join is so that it can be sorted. 
This is based on the og model function used in fulfil/timeout/abort.

I'd rather not change it right now if thats fine with you.